### DISCUSSION TITLE
I can't activate the copilot

### DISCUSSION THREAD

**Discussion by K1R1EIIIKA** [community/community#142840] [GitHub Education] [answered]
### Hi everyone,

Good afternoon, 3 days ago I signed up for a student subscription, it was approved for me, I was given a github pro and a copilot. Today I see a ban on using a copilot and the error:

`"You can't proceed with your payment
There appears to be an issue with the billing information. We're looking into it and will send you an update within 48 hours. Thank you for your patience."` 

How do I get access to the copilot and will it be possible to change the payment information?

**Comment by davevad93** [comment, marked as answer]
Hi @K1R1EIIIKA ,

You should wait for the update from GitHub as indicated in the error message, if you don't receive anything within 48 hours you should contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by Haytham2210** [comment]
You can't proceed with your payment
There appears to be an issue with the billing information. We're looking into it and will send you an update within 48 hours. Thank you for your patience.
